tox.ini

```diff
@@ -0,0 +1,19 @@
+[flake8]
+exclude = .idea,.vscode,.git,pg_data,venv,*/settings/*.py,*/migrations/*.py
+show-source = True
+max-line-length = 100
+application-import-names = smarkets
+max-cognitive-complexity = 15
+ignore = FI1,I100,W503
+
+[coverage:report]
+omit = */reports/constants.py
```

Comment:
Hi,
I think "reports" directory should be renamed as "eaas"....
What do you think?

Reply:
Fixed